### DISCUSSION TITLE
Getting unsupported/unversioned values on renovate logs

### DISCUSSION THREAD

**Discussion by chandapukiran** [renovatebot/renovate#29063] [Request Help]
### What would you like help with?

None

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GHE

### Please tell us more about your question or problem

Hi, Since a week renovate throws an error 
```
DEBUG: Dependency artifactory.xxxxxx/docker-release/common/ubuntu-base/ubuntu has unsupported/unversioned value 2.0.39 

DEBUG: Invalid manifest response (repository=xxxx/xxxxx)
       "registry": "https://artifactory.xxxxxxxxxx/",
       "dockerRepository": "docker-release/common/ubuntu-base/python38",
       "tag": "2.0.47",
       "err": {
         "message": "Schema error",
         "stack": "ZodError: Schema error\n    at Object.get error [as error] (/usr/local/renovate/node_modules/.pnpm/zod@3.23.6/node_modules/zod/lib/types.js:***:31)\n    at DockerDatasource.getManifest (/usr/local/renovate/lib/modules/datasource/docker/index.ts:29***:***6)\n    at processTicksAndRejections (node:internal/process/task_queues:9***:***)\n    at DockerDatasource.getLabels (/usr/local/renovate/lib/modules/datasource/docker/index.ts:4***4:24)\n    at /usr/local/renovate/lib/util/cache/package/decorator.ts:122:18\n    at DockerDatasource.getReleases (/usr/local/renovate/lib/modules/datasource/docker/index.ts:1062:20)\n    at getRegistryReleases (/usr/local/renovate/lib/modules/datasource/index.ts:77:1***)\n    at fetchReleases (/usr/local/renovate/lib/modules/datasource/index.ts:289:1***)",
         "issues": {
           "manifests": {
             "0": {
               "mediaType": "Invalid literal value, expected \"application/vnd.docker.distribution.manifest.v2+json\""
             },
             "1": {
               "mediaType": "Invalid literal value, expected \"application/vnd.docker.distribution.manifest.v2+json\""
             }
           }
         }
       }
```
It has been working since a year without any issues however we have started seeing the above message and no PRs are getting created on any repos to update the base image version. 

We are using artifactory and i can see that we have the new image version and existing images available. 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by viceice** [comment]
maybe related to
- #28970

will look into it now

**Reply by chandapukiran** [reply]
Thanks, I will wait for your suggestion. We are using Renovate 37.358.0

**Reply by viceice** [reply]
- #29065 

this will hopefully show more about the error.

**Reply by chandapukiran** [reply]
Just curious, do I need to use the latest version of Renovate for the above change once your PR is merged.

**Reply by viceice** [reply]
yes.

**Reply by chandapukiran** [reply]
We are using renovate github-action and i think these changes are not updated here : https://github.com/renovatebot/github-action

when i try to fetch the the v37.363.1; i get following 

```
git fetch https://github.com/renovatebot/github-action v37.363.1
fatal: couldn't find remote ref v37.363.1
```

**Reply by HonkingGoose** [reply]
I think the versions of "Renovate the main program" and "Renovate as GitHub Action" _do not match_. The latest version of `renovatebot/github-actions` seems to be `v40.1.11`.

I'm getting confused about the release process and versioning used by `renovatebot/github-action` too. I'm asking for more information in this discussion:

- https://github.com/renovatebot/github-action/discussions/841

I hope to update their readme with the good information later. :smile:
